data/tabular/blood_brain_barrier_martins_et_al/transform.py

```diff
@@ -0,0 +1,235 @@
+            "{SMILES#} represents a molecule that is {penetrate_BBB#not &NULL}{penetrate_BBB__names__adjective}.",  # noqa: E501
+            "{SMILES#} is {penetrate_BBB#not &NULL}{penetrate_BBB__names__adjective}.",
+            "The molecule {SMILES#} is {penetrate_BBB__names__adjective}.",
+            # Instruction tuning text templates
+            """Task: Please classify a molecule based on the description.
+Description: A molecule that is {penetrate_BBB__names__adjective}.
+Molecule {SMILES__description}: {SMILES#}
+Constraint: Even if you are uncertain, you must pick either "True" or "False" without using any other words.
+Result: {penetrate_BBB#False&True}""",
+            """Task: Please classify a molecule based on the description.
+Description: A molecule that is {penetrate_BBB__names__adjective}.
+Molecule {SMILES__description}: {SMILES#}
+Constraint: Answer the question in a complete sentence.
+Result: This molecule is {penetrate_BBB#not &NULL}{penetrate_BBB__names__adjective}.""",
+            """Task: Please create a molecule {SMILES__description} based on the description.
+Description: A molecule that is {penetrate_BBB__names__adjective}.
+Result: {SMILES#}""",
+            # Conversational text templates
+            """User: Can you tell me if the molecule with the {SMILES__description} {SMILES#} is {penetrate_BBB__names__adjective}?
+Assistant: {penetrate_BBB#No&Yes}, this molecule is {penetrate_BBB#not &NULL}{penetrate_BBB__names__adjective}.""",  # noqa: E501
+            """User: Is the molecule with the {SMILES__description} {SMILES#} {penetrate_BBB__names__adjective}?
+Assistant: {penetrate_BBB#No&Yes}, it is {penetrate_BBB#not &NULL}{penetrate_BBB__names__adjective}.""",  # noqa: E501
+            """User: Can you give me the {SMILES__description} of a molecule that is {penetrate_BBB#not &NULL}{penetrate_BBB__names__adjective}?
+Assistant: Yes, here you go: {SMILES#}""",  # noqa: E501
+            """User: I'm looking for the {SMILES__description} of a molecule that is {penetrate_BBB#not &NULL}{penetrate_BBB__names__adjective}?
+Assistant: This is a molecule that is {penetrate_BBB#not &NULL}{penetrate_BBB__names__adjective}: {SMILES#}""",  # noqa: E501
+            """User: I want to create a molecule {SMILES__description}.
+Assistant: This sounds very exciting! Should I consider any constraints during generation?
+User: Yes, please. The molecule should {penetrate_BBB#not &NULL}{penetrate_BBB__names__adjective}.
+Assistant: Ok, this {SMILES__description} is {penetrate_BBB#not &NULL}{penetrate_BBB__names__adjective}: {SMILES#}""",  # noqa: E501
+            """User: I want to create a molecule {SMILES__description}.
+Assistant: This sounds very exciting! Should it be a special molecule?
+User: Yes, the molecule should {penetrate_BBB#not &NULL}be {penetrate_BBB__names__adjective}.
```

Comment:
```suggestion
User: Yes, the molecule should {penetrate_BBB#not &NULL} and be {penetrate_BBB__names__adjective}.
```

Reply:
Your "and" is here too much.
If `penetrate_BBB` is `True` (1) it evals to: `Yes, the molecule should be ...`
If `penetrate_BBB` is `False` (0) it evals to: `Yes, the molecule should not be ...`

Reply:
ah, yes